build/system.gyp

```diff
@@ -11,6 +11,19 @@
   },
   'targets' : [
     {
+      'target_name': 'linux_notification',
+      'type': 'none',
+      'conditions': [
+        ['tizen!=1', {
```

Comment:
My idea was:
(1) For Tizen, the target 'linux_notification' must be visible to avoid gyp error (since Tizen can always include dependencies for Linux in 'xwalk.gyp' )
(2) But for Tizen, the target must don't import 'libnotify'. 
(3) For non-linux platforms, in 'xwalk.gyp' we can make sure they don't include the target 'linux_notification'